geth/jail/extensions/console/console.go

```diff
@@ -0,0 +1,157 @@
+package console
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/robertkrimen/otto"
+	"github.com/status-im/status-go/geth/jail/extensions"
+	"github.com/status-im/status-go/geth/node"
+)
+
+var (
+	// Stdwarn defines the default writer for the console.warn
+	// delivery call.
+	Stdwarn io.Writer = os.Stdout
+
+	// Stdout defines the default writer for the console.error
+	// delivery call.
+	Stdout io.Writer = os.Stdout
```

Comment:
No need to overcomplicate things, `console.log()` is all we need at the moment. We can refine it later, but no real point to do it now. So, I suppose we can remove other writers and let only `log` remain.